### PR TITLE
Bump kubernetes-client-bom from 6.5.1 to 6.6.0

### PR DESCRIPTION
**Pull request by manusa**
Kubernetes Client 6.6.0 was just released: https://github.com/fabric8io/kubernetes-client/releases/tag/v6.6.0

Just speeding up the dependabot process to see if CI reports any issue.

Fixes multiple issues (will try to find the reference later)

/cc @metacosm

### CODE REVIEW

**Review by gastaldi — APPROVED** [review]
LGTM, I'll let @metacosm have the final say before merging this

**Comment by metacosm** [comment]
Should be good on our side.

**Pull request merged**
